R/build-home-community.R

```diff
@@ -0,0 +1,36 @@
+has_contributing <- function(path = ".") {
+  file_exists(path(path, 'CONTRIBUTING.md')) ||
+    file_exists(path(path, '.github', 'CONTRIBUTING.md'))
+}
+
+has_coc <- function(path = ".") {
+  file_exists(path(path, 'CODE_OF_CONDUCT.md')) ||
+    file_exists(path(path, '.github', 'CODE_OF_CONDUCT.md'))
+}
+
+data_home_sidebar_community <- function(pkg){
```

Comment:
```suggestion
data_home_sidebar_community <- function (pkg){
```

This should also call `pkg <- as_pkgdown(path)` to simplify the tests